content/admin/enterprise-management/evacuating-a-cluster-node.md

```diff
@@ -14,40 +14,40 @@ If you're taking a node offline that has any data services (like git, pages, or
 
 1. Find the `uuid` of the node in with the `ghe-config`command.
 
-    ```
+    ```shell
     $ ghe-config cluster._hostname_.uuid
     ```
 
 2. You'll need to monitor the status of your node while the data is being copied. Ideally, the node shouldn't be taken offline until the copying is complete. To monitor the status of your node, run any of the following commands:  
 
     For Git
-    ```
+    ```shell
     ghe-spokes evac-status
     ```
     For {% data variables.product.prodname_pages %}
-    ```
+    ```shell
```

Comment:
This code fence doesn't render `<uuid>`. I think we'll need the raw tags around this code fence too. There are several more examples in this file that will need the same treatment.

Reply:
Interesting, I see that to on the preview site now. Is this actually a bug with the rendering library that I should report upstream? It seems odd that it is altering the content of code blocks as `<code class="hljs language-shell">echo "select count(*) from pages_replicas where host = 'pages-server-<uuid>'" | ghe-dbconsole -y</uuid></code>`